### ISSUE TITLE
Issues running little big mouse alongside fullscreen applications like games

### ISSUE THREAD

**Issue by ObsoletePixel** [mgth/LittleBigMouse#221]
While playing Valorant, LittleBigMouse eats inputs (i.e. clicking with the mouse or using the kayboard sometimes a press will be registered but it'll never see me un-click, locking my input down at inopportune moments.

**Comment by djessy5001** [comment]
It also happens to me while playing League of Legends for instance, when I want to move the camera around (towards my secondary monitor), my mouse jitters.

A possible fix might be an option to disable the application automatically while running fullscreen apps/games.
[https://giphy.com/gifs/littlebigmouseissue221-F21STEMldhO5tFG0RV](url)

**Comment by nathanfohkens** [comment]
Just commenting to bump this as well, an option to disable LittleBihMouse automatically when certain applications are running (or if they're the active window) would be a fantastic feature!

**Comment by Dorecc** [comment]
Wow, I thought I'm really crazy, because I have a slightly malfunctioning mouse but not in a way that would cause random mouse hold downs, yet I couldn't connect the dots for multiple days and I did over the top troubleshooting to find it out, going as far as setting up a separate Windows installation and writing a python script to check if the button is held down on system level.

I couldn't recreate this issue in CSGO but it happened to me when playing Fortnite where you do a lot more large movements in vertical directions compared to CSGO and I finally realized that it happens more often when I try to flick in the direction where the software helps the mouse move, in my case the bottm left (perhaps it's not the exact cause but I was too tired to further test it so it might be any screen edge when flicking)

Also - I used the binary version available in the repo, didn't build it myself

Footage of the tests I did: https://drive.google.com/file/d/1lltcKMIAJP_axZu40NSk5OGZV38IYINf/view

**Comment by Devsider** [comment]
@nathanfohkens Absolutely. Getting rid of those issuses probably wont be easy, but i would be more than happy when it would just disable when certain apps are launched or even just in fullscreen / foreground.

**Comment by MillionLights** [comment]
Eating inputs is a serious issue with this software and it happens to A LOT of games.
It's not intuitive to assume this software is causing it and will send people on a wild goose chase to find the problem or even buy a new mouse, wasting their money.

I noticed this issue first with Grounded™ (Windows Store Version)

There should be a warning in the description raising awareness of this issue.
